pkg/node/address_darwin.go

```diff
@@ -20,11 +20,11 @@ import (
 	"net"
 )
 
-func firstGlobalV4Addr(intf string, preferredIP net.IP) (net.IP, error) {
+func firstGlobalV4Addr(intf string, preferredIP net.IP, preferPublic bool) (net.IP, error) {
```

Comment:
This is interesting, I tried but never be able to compile cilium codebase on my mac due to multiple issues (the best I can do is to use vagrant box :) ). Looking at the implementation for this method, seems like it's just dummy one. 

Anyway, maybe some reference to https://github.com/cilium/cilium/pull/13223 in commit log should be good enough :)

Reply:
maybe because I don't compile entire code base, but rather specific policy package? Sounds good, I'll add 13223 reference in commit message.